snmp/datadog_checks/snmp/data/profiles/_base_cisco_voice.yaml

```diff
@@ -17,14 +17,11 @@ metrics:
           OID: 1.3.6.1.2.1.25.4.2.1.7
         table: hrSWRunTable
         tag: status
-
-  - MIB: HOST-RESOURCES-MIB
-    table:
-      name: hrSWRunTable
-      OID: 1.3.6.1.2.1.25.4.2
-    symbols:
-      - name: hrSWRunStatus
```

Comment:
Why is `hrSWRunStatus` removed?

Reply:
I'm not sure it makes sense to have both as a tag on the previous table and as a metric directly.

Reply:
I originally had it as a metric only, but then added it also as a tag since it was in `hrSWRunPerfTable` and seemed like the most fitting tag there. Would it make sense to keep it as a metric and tag with a different metric here?

Reply:
Possibly! We need to see which entry in that table would make sense for tagging as well then.

Reply:
We could tag with `hrSWRunIndex` or `hrSWRunName`